auth/src/desktop/rpcs/auth_request.cc

```diff
@@ -88,6 +88,7 @@ std::string AuthRequest::GetUrl() {
   } else {
     std::string url(kHttp);
     url += emulator_url;
+    url += "/";
```

Comment:
Just making sure this still works when *not* using emulator?

Reply:
yup verified both scenario.